sql/_hny_join.sql

```diff
@@ -0,0 +1,196 @@
+/*
+DESCRIPTION:
+    Merging devdb with hny. This requires the following procedure.
+    Following _hny.sql query with HNY_matches were completed
+
+    5) Assign flags to indicate one_hny_to_many_dev and/or one_dev_to_many_hny.
+        a) handle one devdb to many hny records cases by concatenating those hny_ids and combines units/fields 
+        to create a single record
+        b) Rest of the cases (one hny to many devdb, many hny to many devdb) are handled by including every 
+        possible joins with devdb records for each hny recor.
+
+    6) Combine the two tables to create HNY_devdb
```

Comment:
Why do directions start with 5 and 6 here? I think the directions themselves are very clear and it makes sense to put this in a new file instead of appending to bottom of existing hny script

Reply:
Good point there. It does rely on the hny_matches created from the last files and that took four steps. That was my reasoning for this weird numbering system.